File: examples/emoji/find/go/findEmoji.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/emoji"
)

// Find an emoji by its alias e.g :beer:
func FindEmoji() {
	emojiService := emoji.NewEmojiService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := emojiService.Find(&emoji.FindRequest{
		Alias: ":beer:",
	})
	fmt.Println(rsp, err)
}

File: examples/emoji/find/node/findEmoji.js
const { EmojiService } = require("m3o/emoji");

// Find an emoji by its alias e.g :beer:
async function findEmoji() {
  let emojiService = new EmojiService(process.env.MICRO_API_TOKEN);
  let rsp = await emojiService.find({
    alias: ":beer:",
  });
  console.log(rsp);
}

findEmoji();
